phys2bids/tests/test_txt.py

```diff
@@ -4,40 +4,42 @@
 from pytest import raises
 import sys
 import io
+import wget
 
 
 def test_read_header_and_channels():
-    test_filename = 'Test_belt_pulse_samefreq.txt'  # this file has a comment and extra tab
+    url = 'https://osf.io/sdz4n/download'
     test_path = resource_filename('phys2bids', 'tests/data')
+    test_filename = 'Test_belt_pulse_samefreq.txt'
     test_full_path = os.path.join(test_path, test_filename)
+    wget.download(url, test_full_path)
     chtrig = 2
     header, channels = txt.read_header_and_channels(test_full_path, chtrig)
     assert len(header) == 16  # check proper header lenght
     assert len(channels) == 1336816  # check proper number of timepoints
     assert len(header[-1]) == 6  # check extra line is deleted
     # load file with comment
-    assert len(channels[6]) == 6  # check the comment has been eliminated
-    test_filename = 'Test_2minRest_trig_multifreq_header_comment.txt'  # this file has a comment and extra tab
+    url = 'https://osf.io/q4x2f/download'
```

Comment:
Same as before, please comment the url corresponds to `test_filename`.

Reply:
done